.pre-commit-config.yaml

```diff
@@ -33,14 +33,14 @@ repos:
         language: fail
         files: '[a-zA-Z0-9_]*/i18n/en\.po$'
   - repo: https://github.com/oca/maintainer-tools
-    rev: e3b46970af62cbbf99f0b45a6675eb4cae744d6d
+    rev: dfba427ba03900b69e0a7f2c65890dc48921d36a
```

Comment:
Looks like you mixed up copier update changes with the migration commit

Reply:
The problem is that such changes are part of the migration commit, so you should undo that.

Reply:
I working in this, sorry

Reply:
Done please review